packages/mjml-core/src/parsers/document.js

```diff
@@ -58,8 +59,14 @@ const mjmlElementParser = (elem, content) => {
 const parseHead = (head, attributes) => {
   const $container = dom.parseHTML(attributes.container)
 
-  each(compact(filter(dom.getChildren(head), child => child.tagName)), element => {
-    const handler = MJMLHeadElements[element.tagName.toLowerCase()]
+  each(compact(filter(dom.getChildren(head), child => child.tagName)), el => {
+    const element = {
+      attributes: dom.getAttributes(el),
+      children: toArray(el.childNodes),
```

Comment:
Hello! The problem with using dom.getChildren is that it returns element.children rather than element.childNodes and element.children doesn't contain text nodes. This breaks <title>I'm a text node</title>.

If changing dom.getChildren to return childNodes won't have any side effects then this would work. I'm off on holiday on the 1st so time is really limited. I'll try and find some time to at least do a test and compare the outputs using the original and then dom.getChildren with childNodes